### ISSUE TITLE
Option to disable logging of keys' ASCII values

### ISSUE THREAD

**Issue by adamrp** [ElDeveloper/keystats#3]
Right now the SQLite database that stores the information stores not only the keystroke counts, but also the ASCII values and timestamps. There should be an option to disable the logging of the keystrokes' ASCII values, since I am concerned about security.

Alternatively, @ElDeveloper suggested that the file could be encrypted, which might also be a acceptable.

**Comment by ElDeveloper** [comment]
I agree with @SantiagoTorres, we need both: encryption and a way to store only counts. Which could perhaps be done with either a new database/table and a new data manager.
